stories/profile/LessonsChallenges.stories.tsx

```diff
@@ -0,0 +1,173 @@
+  { challengeNumber: 4, challengeStatus: 'default' },
+  { challengeNumber: 5, challengeStatus: 'default' },
+  { challengeNumber: 6, challengeStatus: 'default' },
+  { challengeNumber: 7, challengeStatus: 'default' },
+  { challengeNumber: 8, challengeStatus: 'default' },
+  { challengeNumber: 9, challengeStatus: 'default' },
+  { challengeNumber: 10, challengeStatus: 'default' },
+  { challengeNumber: 11, challengeStatus: 'default' },
+  { challengeNumber: 12, challengeStatus: 'default' }
+]
+
+const pendingChallenges = [
+  { challengeNumber: 1, challengeStatus: 'pending' },
+  { challengeNumber: 2, challengeStatus: 'pending' },
+  { challengeNumber: 3, challengeStatus: 'pending' },
+  { challengeNumber: 4, challengeStatus: 'pending' },
+  { challengeNumber: 5, challengeStatus: 'pending' },
+  { challengeNumber: 6, challengeStatus: 'pending' },
+  { challengeNumber: 7, challengeStatus: 'pending' },
+  { challengeNumber: 8, challengeStatus: 'pending' },
+  { challengeNumber: 9, challengeStatus: 'pending' },
+  { challengeNumber: 10, challengeStatus: 'pending' },
+  { challengeNumber: 11, challengeStatus: 'pending' }
+]
+
+const unapprovedChallenges = [
+  { challengeNumber: 1, challengeStatus: 'wrong answer' },
+  { challengeNumber: 2, challengeStatus: 'wrong answer' },
+  { challengeNumber: 3, challengeStatus: 'wrong answer' },
+  { challengeNumber: 4, challengeStatus: 'wrong answer' },
+  { challengeNumber: 5, challengeStatus: 'wrong answer' },
+  { challengeNumber: 6, challengeStatus: 'wrong answer' },
+  { challengeNumber: 7, challengeStatus: 'wrong answer' }
+]
+
+const challengesLesson6 = [
+  { challengeNumber: 1, challengeStatus: 'default' },
+  { challengeNumber: 2, challengeStatus: 'default' },
+  { challengeNumber: 3, challengeStatus: 'default' },
+  { challengeNumber: 4, challengeStatus: 'default' },
+  { challengeNumber: 5, challengeStatus: 'default' },
+  { challengeNumber: 6, challengeStatus: 'default' },
+  { challengeNumber: 7, challengeStatus: 'default' },
+  { challengeNumber: 8, challengeStatus: 'default' }
+]
+
+const challengesLesson5 = [
+  { challengeNumber: 1, challengeStatus: 'default' },
+  { challengeNumber: 2, challengeStatus: 'default' },
+  { challengeNumber: 3, challengeStatus: 'default' },
+  { challengeNumber: 4, challengeStatus: 'default' },
+  { challengeNumber: 5, challengeStatus: 'default' },
+  { challengeNumber: 6, challengeStatus: 'default' },
+  { challengeNumber: 7, challengeStatus: 'default' },
+  { challengeNumber: 8, challengeStatus: 'default' },
+  { challengeNumber: 9, challengeStatus: 'default' }
+]
+
+const challengesLesson9 = [
+  { challengeNumber: 1, challengeStatus: 'default' },
+  { challengeNumber: 2, challengeStatus: 'default' },
+  { challengeNumber: 3, challengeStatus: 'default' },
+  { challengeNumber: 4, challengeStatus: 'default' },
+  { challengeNumber: 5, challengeStatus: 'default' }
+]
+
+const lessons = [
+  {
+    correctAnswers: 10,
```

Comment:
`correctAnswers` does not have to exist. The component should get this value by adding up `challengeStatus: 'passed'` for all the challenges.